tests/bluetooth/tester/src/btp_bap_broadcast.c

```diff
@@ -347,8 +351,11 @@ uint8_t btp_bap_broadcast_source_setup(const void *cmd, uint16_t cmd_len,
 	base_ad.type = BT_DATA_SVC_DATA16;
 	base_ad.data_len = ad_buf.len;
 	base_ad.data = ad_buf.data;
-	err = tester_gap_create_adv_instance(param, BTP_GAP_ADDR_TYPE_IDENTITY, &base_ad, 1, NULL,
-					     0, &gap_settings);
+	sd.type = BT_DATA_NAME_COMPLETE;
+	sd.data_len = strlen(dev_name);
+	sd.data = (const uint8_t *)dev_name;
+	err = tester_gap_create_adv_instance(param, BTP_GAP_ADDR_TYPE_IDENTITY, &base_ad, 1,
```

Comment:
Hmm providing both AD and SD is valid only for instances using legacy PDUs, isin't ?

Reply:
I think you're right, I made a mistake here and on `btp_cap.c`. I'll create a PR to fix that in the afternoon.
Thanks for catching that! 🙂

Reply:
I think some other samples that use ext adv has similar issue,  eg periodic_adv sample where SD data is provided to non-scannable ext adv instance

I'd suggest to re-review those samples where non-legacy instances are used

Reply:
I'll do it, that's a bit sad that not all samples are tested in CI 😅

Reply:
Created the PR: https://github.com/zephyrproject-rtos/zephyr/pull/71848
I looked at the other samples and did not found anything